pkg/postgres/database.go

```diff
@@ -55,10 +55,13 @@ func ParseUsernamePassword(s string) (Credentials, error) {
 // Database ensures that a user with provided password exists on the host and
 // that read and readwrite roles are created with default priviledges on a
 // schema named after the database name.
-func Database(log logr.Logger, db *sql.DB, host string, credentials Credentials) error {
+func Database(log logr.Logger, db *sql.DB, host string, credentials Credentials, managerRole string) error {
```

Comment:
It is not optional from the controllers perspective. What I meant with executing this optimistically is more the fact that we don't want the controller to fail if either the managerRole or the serviceRole turns out not to exist in postgres 🙂

Reply:
We discussed this over the table. What I meant initially was the that from the controllers perspective they should have a list of managers, also so that the list can be empty. But if one of these roles didn't exist in the database the controller would throw an error during the run until it was fixed.

We will start from this point, where the controller requires the input managerRole (at least as far as I can tell) and will continue even if it isn't there.